### DISCUSSION TITLE
Nozzle temp startup

### DISCUSSION THREAD

**Discussion by n6812t** [mriscoc/Ender3V2S1#792] [Q&A]
Hello, using a 3s1 with the firmware update.  Things are a little different, but I got it set up and printing the first print with PLA for a test.

During the startup routine, the bed temp began climbing as expected, but the nozzle temp showed 23/0.  In order to get things started, I ran the Preheat PLA routine, and it worked fine from there.  

Does the startup process work differently in this firmware than in the basic firmware?  Is there a setting I need to make sure the parameter gets passed?

Thank you!

**Comment by n6812t** [comment]
Figured it out.  This firmware does them sequentially!

**Reply by mriscoc** [reply]
Bed takes time to reach thermal stability. So, it is better to do sequentially to give extra time to the bed.

**Comment by TheWebMachine** [comment]
This is controlled by GCode and can be customized in some slicers like Prusaslicer 
Hotend: `M104` - start heating, but don't wait & `M109` - wait for temp before continuing 
Bed: `M140` - start heating, don't wait & `M190` - wait for temp

For example, in Prusaslicer, I have my start Gcode:
* start "pre" heating the bed immediately with M140 (cuz that takes forever) and hotend M104 to 150c, without waiting for them to finish
* then proceed to tilt mesh (UBL) or ABL
* then force a wait for the bed to finish heating with M190
* once the bed hits temp, then push the hotend to first layer temp with M109 to wait for it

**Comment by TheWebMachine** [comment]
But I think you are right. Creality's firmware does not seem to conform to normal Marlin conventions with the bed and hotend temp commands.
